src/context_store/ingestion/pipeline.py

```
@@ -131,6 +131,43 @@ async def dispose(self) -> None:
+        if not source:
+            return 0
+
+        meta = metadata or {}
+
+        # ステップ1: ソースからコンテンツを取得（Adapter 適用）
+        if source_type == SourceType.URL:
+            # URL の場合は実際にフェッチする必要があるため、簡易的な推定は困難
+            # 現時点では RawContent 1件として扱うか、フェッチを許容する
+            try:
+                raw_contents = await self._fetch_url_content(source)
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)